storage/innobase/include/fsp0fsp.h

```diff
@@ -755,6 +755,20 @@ fsp_flags_match(ulint expected, ulint actual)
 	return(actual == expected);
 }
 
+/** Determine if FSP_SPACE_FLAGS are from an incompatible MySQL format.
+@param[in]	flags	the contents of FSP_SPACE_FLAGS
+@return	MySQL flags shifted.
+@retval	0, if not a MySQL incompatible format. >0 */
```

Comment:
I think that the `>0` must be removed.

Please do not add `[in]` in new code. `@param flags` should suffice.